histomicstk/cli/SuperpixelSegmentation/CreateDataset.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<executable>
+  <category>HistomicsTK</category>
+  <title>Create HistomicsML Dataset</title>
+  <description>Create HistomicsML files</description>
+  <version>0.1.0</version>
+  <documentation-url>https://digitalslidearchive.github.io/HistomicsTK/</documentation-url>
+  <license>Apache 2.0</license>
+  <contributor>Deepak Roy Chittajallu (Kitware), Sanghoon Lee (Emory University)</contributor>
+  <acknowledgements>This work is part of the HistomicsTK project.</acknowledgements>
+  <parameters>
+    <label>IO</label>
+    <description>Input/output parameters</description>
+    <string>
+      <name>inputSlidePath</name>
+      <label>Input Images</label>
+      <description>Input images</description>
+      <channel>input</channel>
+      <index>0</index>
+    </string>
+    <string>
+      <name>inputPCAModel</name>
+      <label>Input PCA fitted model</label>
+      <description>Input PCA Model</description>
+      <channel>input</channel>
+      <index>1</index>
+    </string>
+    <file fileExtensions=".h5">
+      <name>outputDataSetFile</name>
+      <label>Output DataSet file</label>
+      <channel>output</channel>
+      <index>2</index>
+      <description>Output dataset file (*.h5)</description>
+      <default>spfeatures.h5</default>
+    </file>
+    <file fileExtensions=".txt">
+      <name>outputBoundariesFile</name>
+      <label>Output Boundary file</label>
+      <channel>output</channel>
+      <index>3</index>
+      <description>Output boundary file (*.txt)</description>
+      <default>spboundaries.txt</default>
+    </file>
+  </parameters>
+  <parameters advanced="true">
+    <label>Color Normalization</label>
+    <description>Color Normalization parameters</description>
+    <double-vector>
+      <name>reference_mu_lab</name>
+      <label>Reference Mean LAB</label>
+      <description>Mean of reference image in LAB color space for Reinhard color normalization</description>
+      <longflag>reference_mu_lab</longflag>
+      <default>8.63234435, -0.11501964, 0.03868433</default>
+    </double-vector>
+    <double-vector>
+      <name>reference_std_lab</name>
+      <label>Reference Stddev LAB</label>
+      <description>Standard deviation of reference image in LAB color space for Reinhard color normalization</description>
+      <longflag>reference_std_lab</longflag>
+      <default>0.57506023, 0.10403329, 0.01364062</default>
+    </double-vector>
+    <double-vector>
+      <name>source_mu_lab</name>
+      <label>Source Mean LAB</label>
+      <description>Mean of source image in LAB color space for Reinhard color normalization</description>
+      <longflag>source_mu_lab</longflag>
+      <default>8.96447078, -0.0879669, 0.04412108</default>
```

Comment:
Source mu and std in Lab is supposed to computed from the slide being processed.

You can provide these as parameters but then the default values should be some None value like -1, -1, -1. If these parameters are not supplied then it should compute them on the fly.